Tests/govuk_ios/govuk_ios_snapshot_tests/Helpers/SnapshotTestCase.swift

```diff
@@ -0,0 +1,121 @@
+import UIKit
+
+import iOSSnapshotTestCase
```

Comment:
Is there an approval process required to use this? I think if we are using it we will need to include the MIT licence.

Reply:
We are going to need to include all 3rd party libs licences and acknowledgements, yeah. Not sure we will need much approval to use it in general given its not production code. Worth checking before merging though.